contrib/wheels/ci.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+cd smt-switch/
+mkdir -p build
+pip install Cython==0.29 --install-option="--no-cython-compile"
```

Comment:
Lets actually not use the `--no-cython-compile` option. It speeds up the build but then the bindings are less performant. For the release it should be more optimized.